README.md

```diff
@@ -1,7 +1,333 @@
+
 # sb-util
-**sb-util** is a utility that allows developers to query a Scratch Project for collections of sprites, blocks, assets, and any other information within a *.sb3 file's project.json.
 
-This repo holds the TypeScript implementation of sb-util, which can be imported as a JavaScript library or used as a CLI.
+## Table of Contents
+- [sb-util Proposal (RFC)](https://github.com/bocoup/sb-util/tree/readme-api-proposal#sb-util-proposal-rfc)
+  - [JavaScript API Proposal](https://github.com/bocoup/sb-util/tree/readme-api-proposal#sb-util-proposal-rfc)
+  - [CLI Proposal](https://github.com/bocoup/sb-util/tree/readme-api-proposal#sb-util-proposal-rfc)
+- [Development](https://github.com/bocoup/sb-util/tree/readme-api-proposal#development)
+
+---
+
+## sb-util Proposal (RFC)
+
+We, at Bocoup, propose **sb-util**, a JavaScript and CLI utility that allows developers to query a Scratch Project for collections of sprites, blocks, assets, and project metadata.
+
+sb-util will accomplish this by consuming ***.sb3** files generate by Scratch. **.sb3** files are used to save and load projects, and within an *.sb3 file there is a **project.json**, which is the JSON representation of the entities within a Scratch project. sb-util will provide an API that allows developers to query the project.json for project information.
```

Comment:
check formatting here you have too many `***` somewhere

Reply:
Nice catch! Will fix